jsk_data/launch/common_record.launch

```diff
@@ -11,7 +11,8 @@
   <arg name="save_multisense" default="false" />
   <arg name="camera_namespace" default="openni"/>
   
-  <arg name="save_dir" default="$(env HOME)/.ros"/>
+  <arg name="save_dir"/> <!-- destination directory of bagfile -->
```

Comment:
By default `rosbag record` records bag file on `cwd`.
This argument implicitly sets save destination to `$HOME/.ros`, and this can be dangerous for machine which has limited space.
(e.g. PR2 has external HDD for recording bag files, but `$HOME` which is mounted as system HDD has only 1~10GB and this is shared with system region, so it may occurs failure of booting next time.
Anyway it's better to be another PR for this part. So I reverted this time.